Add Additional Docs on export tool (#825)
gravwell/wiki: tools/tools.md

-rw-r--r-- 1 gravwell gravwell 3494587 Jun 28 14:31 2023-06-28-10:18:40.json
```

### Output Data Structure

> The export tool outputs data in a JSON structure that is to be easy to work with by a variety of tools; the native golang types are documented on our open source repo [here](https://pkg.go.dev/github.com/gravwell/gravwell/v3@v3.8.24/client/types#SearchEntry) and [here](https://pkg.go.dev/github.com/gravwell/gravwell/v3@v3.8.24/client/types#EnumeratedPair):

Reviewer: ```suggestion
The export tool outputs data in a JSON structure which is easy to work with using a variety of tools; the native golang types are documented on our open source repo [here](https://pkg.go.dev/github.com/gravwell/gravwell/v3@v3.8.24/client/types#SearchEntry) and [here](https://pkg.go.dev/github.com/gravwell/gravwell/v3@v3.8.24/client/types#EnumeratedPair):
```
